src/services/codefixes/convertToTypeOnlyImport.ts

```diff
@@ -55,9 +57,12 @@ function doChange(changes: textChanges.ChangeTracker, sourceFile: SourceFile, de
     else {
         const importClause = declaration.importClause as ImportClause;
         if (importClause.name && importClause.namedBindings) {
+            const modifiers = getSynthesizedDeepClones(declaration.modifiers, /*includeTrivia*/ true);
+            const moduleSpecifier = getSynthesizedDeepClone(declaration.moduleSpecifier, /*includeTrivia*/ true);
+            const assertClause = getSynthesizedDeepClone(declaration.assertClause, /*includeTrivia*/ true);

```

Comment:
I *think* that even for cloned nodes, these can't be shared. So each import needs to get its own clones of the `modifiers`, the `moduleSpecifier`, and the `assertClause`.